ci/test_wheels.py

```diff
@@ -0,0 +1,58 @@
+import glob
+import os
+import platform
+import shutil
+import subprocess
+import sys
+
+py_ver = platform.python_version()
+is_32_bit = os.getenv("IS_32_BIT") == "true"
+try:
+    wheel_dir = sys.argv[1]
+    wheel_path = glob.glob(f"{wheel_dir}/*.whl")[0]
+except IndexError:
+    # Not passed
+    wheel_path = None
+print(f"IS_32_BIT is {is_32_bit}")
+print(f"Path to built wheel is {wheel_path}")
+if os.name == "nt":
+    if is_32_bit:
+        sys.exit(0)  # No way to test Windows 32-bit(no docker image)
+    if wheel_path is None:
+        raise ValueError("Wheel path must be passed in if on 64-bit Windows")
+    print(f"Pulling docker image to test Windows 64-bit Python {py_ver}")
+    subprocess.run(f"docker pull python:{py_ver}-windowsservercore", check=True)
```

Comment:
Might also be worth just using the `docker` python package rather than running all of these in subprocess commands

https://pypi.org/project/docker/

Reply:
I don't know if there will be too much improvement here(have not used that package myself). Looking at the examples, though, (e.g. ``client.containers.run("ubuntu:latest", "echo hello world")``), it seems like the syntax is very similar to a subprocess call.